docs/source/release/v6.2.0/diffraction.rst

```diff
@@ -5,110 +5,124 @@ Diffraction Changes
-- improve performance of :ref:`ApplyDiffCal <algm-ApplyDiffCal>` on large instruments eg WISH. This in turn improves the performance of :ref:`AlignAndFocusPowder <algm-AlignAndFocusPowder>`
-- :ref:`LoadILLPolarizedDiffraction <algm-LoadILLPolarizedDiffraction>` now sorts the polarization orientations and enforces spin-flip, then non-spin-flip order
-- :ref:`PolDiffILLReduction <algm-PolDiffILLReduction>` received a number of improvements: changes names of input workspaces to contain polarization information,
-  transmission can be provided as a number or a workspace group, new data averaging option depending on measurement 2theta, option to display all measured points
-  on a scatter plot, new option for self-attenuation treatment using measured tranmission.
-- added a 3mf format file describing the PEARL sample and environment shapes for the P-E press. Also fixed a couple of minor issues in the 3mf file format loader used in ref:`LoadSampleEnvironment  <algm-LoadSampleEnvironment>`
-- :ref:`LoadILLDiffraction <algm-LoadILLDiffraction>` now adds input run number also to a metadata field `run_list`, indended to contain a full list of numors, handled by :ref:`MergeRuns <algm-MergeRuns>`
-- :ref:`LoadWANDSCD <algm-LoadWANDSCD-v1>` now has a new option to perform normalization in the same loading process.
+* :ref:`AlignAndFocusPowder <algm-AlignAndFocusPowder-v1>` permits masking of discrete wavelength ranges to zero, for resonance filtering.
+* Improved performance of :ref:`ApplyDiffCal <algm-ApplyDiffCal>` on large instruments e.g. WISH. This in turn improves the performance of :ref:`AlignAndFocusPowder <algm-AlignAndFocusPowder>`.
+* :ref:`ConvertDiffCal <algm-ConvertDiffCal-v1>` now optionally updates a previous calibration when converting offsets.
+* :ref:`LoadILLDiffraction <algm-LoadILLDiffraction>` now adds input run number also to a metadata field `run_list`, intended to contain a full list of numbers, handled by :ref:`MergeRuns <algm-MergeRuns>`.
+* :ref:`LoadILLPolarizedDiffraction <algm-LoadILLPolarizedDiffraction>` now sorts the polarization orientations and enforces spin-flip, then non-spin-flip order.
+* :ref:`LoadWANDSCD <algm-LoadWANDSCD-v1>` has a new option to perform normalization in the same loading process.
+* :ref:`PDCalibration <algm-PDCalibration-v1>` has a new option to use the :ref:`IkedaCarpenterPV <func-IkedaCarpenterPV>` peak function.
+* :ref:`PolDiffILLReduction <algm-PolDiffILLReduction>` received a number of improvements
+
+  * Changes names of input workspaces to contain polarization information.
+  * Transmission can be provided as a number or a workspace group.
+  * New data averaging option depending on measurement 2theta.
+  * Option to display all measured points on a scatter plot.
+  * New option for self-attenuation treatment using measured transmission.
+
+* Several improvements have been made to the group calibration routine including
```

Comment:
Looks like there's a difference in formatting between the sub-bullet points lists on these two release notes 
![image](https://user-images.githubusercontent.com/55979119/131997619-cfc6f5b1-2256-47d6-9491-5d065f60f3c9.png)